LayoutTests/imported/w3c/web-platform-tests/xhr/send-network-error-sync-events.sub.htm

```diff
@@ -17,7 +17,7 @@
         {
             var xhr = new XMLHttpRequest();
 
-            xhr.open("POST", "http://nonexistent.{{host}}:{{ports[http][0]}}", false);
+            xhr.open("POST", "http://{{host}}:3", false); // Bad port.
```

Comment:
3 is not a bad port as far as I can tell from https://fetch.spec.whatwg.org/#bad-port.